### PR TITLE
Fix flaky filling SKU and verifying CYS footer e2e tests

### LINKED ISSUE #49109: [Flaky] merchant/products/block-editor/product-inventory-block-editor.spec.js
can update sku 

Failed on: https://github.com/woocommerce/woocommerce/actions/runs/9780322159

### PR DESCRIPTION
**Pull request by veljkho**
### Submission Review Guidelines:

-   I have followed the [WooCommerce Contributing Guidelines](https://github.com/woocommerce/woocommerce/blob/trunk/.github/CONTRIBUTING.md) and the [WordPress Coding Standards](https://make.wordpress.org/core/handbook/best-practices/coding-standards/).
-   I have checked to ensure there aren't other open [Pull Requests](https://github.com/woocommerce/woocommerce/pulls) for the same update/change.
-   I have reviewed my code for [security best practices](https://developer.wordpress.org/apis/security/).
-   Following the above guidelines will result in quick merges and clear and detailed feedback when appropriate.

### Changes proposed in this Pull Request:

Closes #49109 #49080

For the footer test, it is very hard to fix better without completely refactoring the test.
For the inventory test, waiting for response solves it.

### How to test the changes in this Pull Request:

Using the [WooCommerce Testing Instructions Guide](https://github.com/woocommerce/woocommerce/wiki/Writing-high-quality-testing-instructions), include your detailed testing instructions:

```
pnpm test:e2e-pw tests/e2e-pw/tests/customize-store/assembler/footer.spec.js
```

```
pnpm test:e2e-pw tests/e2e-pw/tests/merchant/products/block-editor/product-inventory-block-editor.spec.js
```

### Changelog entry

-   [ ] Automatically create a changelog entry from the details below.

-   [ ] This Pull Request does not require a changelog entry. (Comment required below)

<details>

<summary>Changelog Entry Details</summary>

#### Significance

-   [ ] Patch
-   [ ] Minor
-   [ ] Major

#### Type

-   [ ] Fix - Fixes an existing bug
-   [ ] Add - Adds functionality
-   [ ] Update - Update existing functionality
-   [ ] Dev - Development related task
-   [ ] Tweak - A minor adjustment to the codebase
-   [ ] Performance - Address performance issues
-   [ ] Enhancement - Improvement to existing functionality

#### Message 

</details>

<details>

<summary>Changelog Entry Comment</summary>

#### Comment 

</details>

### CODE REVIEW

File: plugins/woocommerce/tests/e2e-pw/tests/customize-store/assembler/footer.spec.js [hunk 1]
```diff
@@ -146,8 +146,8 @@ test.describe( 'Assembler -> Footers', { tag: '@gutenberg' }, () => {
 			.all();
 
 		for ( const footerPicker of footerPickers ) {
-			await footerPicker.waitFor();
-			await footerPicker.click();
+			await footerPicker.waitFor( { state: 'visible' } );
```
Comment (adimoldovan) [inline plugins/woocommerce/tests/e2e-pw/tests/customize-store/assembler/footer.spec.js]: State visible is the default, so no need be explicit.

File: plugins/woocommerce/tests/e2e-pw/tests/customize-store/assembler/footer.spec.js [hunk 2]
```diff
@@ -146,8 +146,8 @@ test.describe( 'Assembler -> Footers', { tag: '@gutenberg' }, () => {
...
+			await footerPicker.click( { delay: 300 } );
```
Comment (adimoldovan) [inline plugins/woocommerce/tests/e2e-pw/tests/customize-store/assembler/footer.spec.js:150]: Why is the test failing? I'm trying to understand why adding a delay would help.

Comment (veljkho) [reply plugins/woocommerce/tests/e2e-pw/tests/customize-store/assembler/footer.spec.js:150]: I think it's a race condition. The test is very flaky these days, and I used the fastest solution. The preview page behaves differently due to very fast PW. There were no responses to wait for.

Comment (veljkho) [reply plugins/woocommerce/tests/e2e-pw/tests/customize-store/assembler/footer.spec.js:150]: I'm going to rerun core e2e tests 1/6 shard a few times to see if it solves flakiness

Comment (veljkho) [reply plugins/woocommerce/tests/e2e-pw/tests/customize-store/assembler/footer.spec.js:150]: Worked great in CI after rerunning 5-6 times

**Review by adimoldovan — APPROVED** [review]

**Comment by veljkho** [comment]
So far so good, it's [green after the PR's merge](https://buildkite.com/organizations/automattic/analytics/suites/woocommerce-core-e2e-tests/tests/018fa08c-e765-7c11-9664-1cfe33138811?branch=all+branches).

**Pull request merged**